lib/metasploit_data_models/version.rb

```diff
@@ -11,6 +11,8 @@ module Version
     MINOR = 2
     # The patch version number, scoped to the {MAJOR} and {MINOR} version numbers.
     PATCH = 5
```

Comment:
I'll correct it on the merge, but in general, you need to increment PATCH and set PRERELEASE if you were going to prerelease this gem on rubygems.org for testing as 1.2.5-recog-service-never-match < 1.2.5, so with just setting PRERELEASE you're saying this PR is < 1.2.5, which has already been released.  1.2.5 < 1.2.6-recog-service-never-match < 1.2.6 is the proper ordering.